Gemfile.lock

```diff
@@ -250,25 +245,25 @@ GEM
     method_source (0.9.2)
     mime-types (3.3)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.0904)
+    mime-types-data (3.2019.1009)
     mimemagic (0.3.3)
     mini_magick (4.9.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.12.2)
+    minitest (5.13.0)
     msgpack (1.3.1)
-    mysql2 (0.5.2)
+    mysql2 (0.5.3)
     net-scp (2.0.0)
       net-ssh (>= 2.6.5, < 6.0.0)
     net-ssh (5.2.0)
     netrc (0.11.0)
     nio4r (2.5.2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
-    oj (3.9.1)
+    oj (3.10.0)
```

Comment:
### [oj](http://www.ohler.com/oj/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.9.1 → 3.10.0

[_[change-log](https://github.com/ohler55/oj/blob/master/CHANGELOG.md), [source-code](https://github.com/ohler55/oj)_]





<details>
<summary>Commits</summary>

A change of **20** commits. See the full changes on [the compare page](https://github.com/ohler55/oj/compare/v3.9.1...v3.10.0).

These are the first 10 commits:
- (3fd3f83) [Travis version work around](https://github.com/ohler55/oj/commit/3fd3f83d7d7b51a55278d8944d3894fe6d3afb97)
- (b9af4b6) [Travis version work around](https://github.com/ohler55/oj/commit/b9af4b6c861728414dcb627c5d59bf6a3d4f7621)
- (dbf7dec) [Mimic parse error (#556)](https://github.com/ohler55/oj/commit/dbf7dec57fea45086f66bb92366c41d41350a22d)
- (cd9aae6) [Fix comments to reflect correct error class JSON::ParserErro…](https://github.com/ohler55/oj/commit/cd9aae6ae7b1d73744e1ddf9cdc79138ab0cb058)
- (9ae66cd) [Use Rails 5.2.3 test cases (#558)](https://github.com/ohler55/oj/commit/9ae66cd56a0166e6dee6909a49488cdfc0d5a607)
- (b635355) [Add Rails 6.0.0 test cases (#562)](https://github.com/ohler55/oj/commit/b635355119f0166f7e697e907410310f25381e4a)
- (addf060) [Visually differentiate the build badges [ci skip] (#563)](https://github.com/ohler55/oj/commit/addf0606c03173562d39b5b00349f06a60b61993)
- (101e2be) [Create SECURITY.md](https://github.com/ohler55/oj/commit/101e2be122001080a520fb85cfd30642ec24e49f)
- (1ac1f9f) [Update README.md](https://github.com/ohler55/oj/commit/1ac1f9f670fb6478ee2ec7e23e4bc62d6053fa27)
- (ce16cba) [Fix load_file to honor custom mode](https://github.com/ohler55/oj/commit/ce16cbac7d861927cf6f97c09ffab70f11bdb24b)


</details>